RELEASE_NOTES.md

```diff
@@ -1,13 +1,16 @@
-# Cosmos SDK v0.44.3 Release Notes
+# Cosmos SDK v0.44.4 Release Notes
 
 This release introduces bug fixes and improvements on the Cosmos SDK v0.44 series.
 
-The main performance improvement concerns gRPC queries, which are now able to run concurrently on the node ([\#10045](https://github.com/cosmos/cosmos-sdk/pull/10045)). To benefit from this performance boost, make sure to send your gRPC queries to the gRPC server directly (default port `9090`) instead of using the Tendermint RPC [`abci_query` endpoint](https://docs.tendermint.com/master/rpc/#/ABCI/abci_query) (default port `26657`).
+v0.44.0 x/auth migration has a **vesting account bug**, which vanishes `delegated_vesting` field from `BaseVestingAccount`. Recovery, unfortunately, is complicated and either involves manually overwriting it or querying an old state.
+We had to change the order of module migration by pushing x/auth to the end. Auth module state depends on x/stake and should be run last. We have updated the documentation to provide more details how to change module migration order. This is technically a breaking change, but only impacts updates between the major version change, hence migrating from the previous patch release (0.44.x) doesn't cause new migration and doesn't break the state.
 
-This release notably also:
+Other bug fixes:
++ query account balance by ibc denom
```

Comment:
This is actually a client-breaking change for HTTP clients. We should document that better:
```diff
- /bank/v1beta1/balances/{address}/{ibc_denom}
+ /bank/v1beta1/balances/{address}/by_denom?denom={ibc_denom}
```

Reply:
OK, I've added more details. I'm not sure if we should link PRs here - these are in the changelog.